app/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.kt

```diff
@@ -348,7 +348,9 @@ open class FolderPickerActivity :
     }
 
     private fun toggleChooseEnabled() {
-        mChooseBtn?.isEnabled = checkFolderSelectable()
+        val isFolderSelectable = checkFolderSelectable()
+        mChooseBtn?.isEnabled = isFolderSelectable
+        mChooseBtn?.alpha = if (isFolderSelectable) 1f else 0.5f
```

Comment:
@tobiasKaminsky We have different behaviour for disabled state. 
Let me know if this also has to be upstreamed.